packages/config-vite-base/package.json

```diff
@@ -10,6 +10,19 @@
   "scripts": {
     "build": "tsup"
   },
+  "exports": {
```

Comment:
Was facing issues with our new configs, adding the exports and adding cjs fixed the issues,